content/ja/post/tem-hca/index.md

```diff
@@ -0,0 +1,47 @@
+---
+# Documentation: https://sourcethemes.com/academic/docs/managing-content/
+
+title: "Thonglek君の国際会議IEICE-EE2021での発表"
+summary: ""
+authors: ["kundjanasith-thonglek"]
+tags: ["Machine Learning"]
+categories: []
+date: 2021-05-27T10:29:13+09:00
+
+# Optional external URL for project (replaces project detail page).
+external_link: ""
+
+# Featured image
+# To use, add an image named `featured.jpg/png` to your page's folder.
+# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
+image:
+  caption: ""
+  focal_point: ""
+  preview_only: false
+
+# Custom links (optional).
+#   Uncomment and edit lines below to show custom links.
+# links:
+# - name: Follow
+#   url: https://twitter.com
+#   icon_pack: fab
+#   icon: twitter
+
+url_code: ""
+url_pdf: ""
+url_slides: ""
+url_video: ""
+
+# Slides (optional).
+#   Associate this project with Markdown slides.
+#   Simply enter your slide deck's filename without extension.
+#   E.g. `slides = "example-slides"` references `content/slides/example-slides.md`.
+#   Otherwise, set `slides = ""`.
+slides: ""
+---
+
+当研究室D1のKundjanasith Thonglek (Tem) 君が，機械学習の応用に関する国際会議
+[Technical Meeting on Home and Consumer Appliances (IEICE-EE) 2021](https://www.ieice.org/ken/program/index.php?mode=program&tgs_regid=da7e1616e5f2257e688d8bf46f549def5a7a9e50dfa313bfb258a79e10aed93a&tgid=IEE-HCA&layout=&lang=eng)
+にて下記の論文を発表しました:
+
+> Kundjanasith Thonglek, Kohei Ichikawa, Kazufumi Yuasa, Tadatoshi Babasaki, "LSTM-based Neural Network Model for Predicting Solar Power Generation", Technical Meeting on Home and Consumer Appliances (IEICE-EE) 2021, May, 2021 
```

Comment:
"Technical Meeting on Home and Consumer Appliances (IEICE-EE) 2021, May, 2021"
->
"Technical Committee on Energy Engineering in Electronics and Communications (IEICE-EE), 2021 May"